gnmi_server/server.go

```diff
@@ -376,6 +382,18 @@ func (s *Server) Get(ctx context.Context, req *gnmipb.GetRequest) (*gnmipb.GetRe
 	return &gnmipb.GetResponse{Notification: notifications}, nil
 }
 
+// SaveOnSetEnabled saves configuration to a file
+func SaveOnSetEnabled() {
+	if err := transformer.SaveStartupConfig(); err != nil {
```

Comment:
```
gnmi_server/server.go:387:12: undefined: transformer.SaveStartupConfig
```

Reply:
Its dependent on https://github.com/sonic-net/sonic-mgmt-common/pull/83

Reply:
If I understand correctly, this would require a much deeper change to support. I've currently modified according to anand's and sachin's request to place SaveStartupConfig in translib instead of transformer.

Reply:
Implemented translib rpc for copy.